drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -110,8 +128,8 @@ static void nrf5_get_eui64(uint8_t *mac)
 	}
```

Comment:
I can not add comment to the line that is outside of change scope so adding here:
in line 121: you pass EUI64_ADDR directly that points to the start of the UICR/OTP array. Shouldn't an offcest be added there to point to the register specified by the CONFIG_IEEE802154_UICR_EUI64_REG?

Reply:
You are right, will fix.